packages/test/test-end-to-end-tests/src/test/deRehydrateContainerTests.spec.ts

```diff
@@ -33,6 +29,8 @@ import { IRequest, IFluidCodeDetails } from "@fluidframework/core-interfaces";
 import { requestFluidObject } from "@fluidframework/runtime-utils";
 import { ChildLogger } from "@fluidframework/telemetry-utils";
 import { describeFullCompat } from "@fluidframework/test-version-utils";
+// eslint-disable-next-line import/no-internal-modules
+import { getSnapshotTreeFromSerializedContainer } from "@fluidframework/container-loader/dist/utils";
```

Comment:
Please take a glance if tests relying on this API are in the right place, i.e. should they be moved into fluidframework/container-loader itself if this is possible? Import suggests they are in the wrong place, though I have not looked into how hard it is to move / recreate them in container-loader package.

Reply:
Actually want to test the contents in actual tree. So that is why using in end to end tests. We also a few test as unit tests. So that is why importing like an internal module here.